# Type: refactor
# Subtype: R3
# English Summary: Change the access modifier of LogItem class to public
# Chinese Summary: 更改LogItem类的访问权限修饰符为public
# Entry Number: 26

# === Git Diff Content ===

diff --git a/src/main/java/com/xtremelabs/robolectric/ShadowLogger.java b/src/main/java/com/xtremelabs/robolectric/ShadowLogger.java
index da3dfec..d311c87 100644
--- a/src/main/java/com/xtremelabs/robolectric/ShadowLogger.java
+++ b/src/main/java/com/xtremelabs/robolectric/ShadowLogger.java
@@ -83,7 +83,7 @@ public class ShadowLogger {
         debug, error, info, verbose, warning, wtf
     }
 
-    static class LogItem {
+    public static class LogItem {
         public final LogType type;
         public final String tag;
         public final String msg;
